Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1495,6 +1495,75 @@ def messages() -> Response:
     return Response(status=200)
 
 
+def ring_user():
+    """Rings a user on Teams.
+
+    Notes:
+        This is a ring only! no media plays in case the generated call is answered.
+
+    Returns:
+        None.
+    """
+    bot_id = demisto.params().get('bot_id')
+    integration_context: dict = demisto.getIntegrationContext()
+    tenant_id: str = integration_context.get('tenant_id', '')
+    if not tenant_id:
+        raise ValueError(
+            'Did not receive tenant ID from Microsoft Teams, verify the messaging endpoint is configured correctly.'
+        )
+    # get user to call name and id
+    username_to_call = demisto.args().get('username')
+    users: list = get_users()
+    user_id: str = str()
+    for user in users:
+        if username_to_call in {user.get('displayName', ''), user.get('mail'), user.get('userPrincipalName')}:
+            user_id = user.get('id', '')
+            break
+    if len(user_id) == 0:
+        raise ValueError(f'User {username_to_call} was not found')
+
+    call_request_data = {
+        "@odata.type": "#microsoft.graph.call",
+        "callbackUri": 'https://callback.url',
+        "direction": "outgoing",
+        "source": {
+            "@odata.type": "#microsoft.graph.participantInfo",
+            "identity": {
+                "@odata.type": "#microsoft.graph.identitySet",
+                "application": {
+                    "@odata.type": "#microsoft.graph.identity",
+                    "id": bot_id
+                }
+            }
+        },
+        "targets": [
+            {
+                "@odata.type": "#microsoft.graph.invitationParticipantInfo",
+                "identity": {
+                    "@odata.type": "#microsoft.graph.identitySet",
+                    "user": {
+                        "@odata.type": "#microsoft.graph.identity",
+                        "displayName": username_to_call,
+                        "id": user_id
+                    }
+                }
+            }
+        ],
+        "requestedModalities": [
+            "audio"
+        ],
+        "mediaConfig": {
+            "@odata.type": "#microsoft.graph.serviceHostedMediaConfig",
+        },
+        "tenantId": tenant_id
+    }
+    response = http_request(method='POST',
```

Comment:
we should split this func to two: command and request
you are not supposed to call `http_request` from a command func